tools/unicodeCat.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const request = require('request');
+
+create(
+    'bmpLetters',
+    'https://raw.githubusercontent.com/mathiasbynens/unicode-6.2.0/' +
```

Comment:
Not sure I like the external reference here. Can't we include this in our codebase in the `tools` directory?